Makefile

```diff
@@ -0,0 +1,21 @@
+SNAPPY = $(CHAIN)/vendor/github.com/google/snappy
+ROCKSDB = $(CHAIN)/vendor/github.com/facebook/rocksdb
+
+null:
+	@echo Please specify a make target.
+
+## builds chain core with c dependencies, for development environments
+build-dev: build-lib
+	go install -tags 'localhost_auth http_ok reset' chain/cmd/cored
```

Comment:
That seems like a real edge case, but I don't see any harm in protecting against it!